tests/Domain/StagerTest.php

```diff
@@ -132,4 +177,40 @@ public function providerCommandContainsWorkingDirOption(): array
             [['-d' => 'lorem/ipsum']],
         ];
     }
+
+    public function testProcessFailed(): void
```

Comment:
This method and the next are confusing too. It'd be helpful if the doc comments explained exactly what the expectations are and what we're testing.

Reply:
Okay. I can do that.

Reply:
Actually, same as before: I use the same pattern--usually even the same methods names--all over the place, so I'm leaving it here.